file-formats/sfpf/sfpf-v1.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/sfpf/sfpf-v1.json",
+  "title": "Object Type SFPF",
+  "description": "Object type SFPF",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "Format Version",
+      "description": "Format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the Form Template",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original language",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP cloud development"
+          ],
+          "default": "cloudDevelopment"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "originalLanguage"
+      ]
+    },
+    "props": {
+      "title": "Properties",
+      "description": "Form Template Properties",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Name",
+          "description": "Name of the Form Template",
+          "type": "string",
+          "maxLength": 30
+        },
+        "dataprovider": {
+          "title": "Dataprovider",
+          "description": "RAP Service Definition that implements the business context for this form template",
+          "type": "string",
+          "maxLength": 40
+        },
+        "embed": {
+          "title": "Font Embed",
+          "description": "Automatically embed font files into the output. Useful if your output uses fonts that are not delivered by default, for asian fonts or to include your own branding.",
+          "type": "boolean"
+        },
+        "mirroring": {
+          "title": "Mirror in RTL",
```

Comment:
Did you regenerate the schema based on your interface. It seems some descriptions were not taken over.

This is just an example. But in your interface you defined it as follows

```
"! <p class="shorttext">Mirroring</p>
```

In the JSON schema it's "Mirror in RTL".

This is also the reason for the failing test:

```
SFPF
diff --strip-trailing-cr generated/sfpf-v1.json ../file-formats/sfpf/sfpf-v1.json
80,81c80,81
<           "title": "Mirroring",
<           "description": "When your form template is designed in an LTR language and your target output is an RTL language, should fields be automatically mirrord in the output?",
---
>           "title": "Mirror in RTL",
>           "description": "When your form template is designed in an LTR language and your target output is an RTL language, should fields be automatically be mirrord in the output?",
85,86c85,86
<           "title": "Optimize",
<           "description": "Automatically analyze the data binding of the template and only select necessary fields from the business data input of the FDP",
---
>           "title": "FDP Data Optimization",
>           "description": "When your form template is designed in an LTR language and your target output is an RTL language, should fields be automatically be mirrord in the output?",
```